metricbeat/module/kibana/cluster_rules/data.go

```diff
@@ -0,0 +1,123 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cluster_rules
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/elastic/beats/v7/metricbeat/helper/elastic"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+
+	s "github.com/elastic/beats/v7/libbeat/common/schema"
+	c "github.com/elastic/beats/v7/libbeat/common/schema/mapstriface"
+	"github.com/elastic/beats/v7/metricbeat/mb"
+)
+
+var (
+	kibanaSchema = s.Schema{
```

Comment:
Is this `kibanaSchema` the same everywhere? If so, it should be in the base `kibana` library and exported.

This also looks suspiciously similar to a lot of the code in `kibana/status/data.go`, we may want to just export/refactor that code and use it here, but that might fall outside the scope of this PR

Reply:
Is there an example of "exporting" a variable? I see patterns for functions and I get certainly define a function that returns the schema but unsure if it's better to do it as a variable instead

Reply:
I'm not quite sure what you're asking, but to clarify, I was thinking that `kibanaSchema` seemed to be copied across multiple places, so it might be best to define it in `module/kibana/kibana.go` or something as a public variable like `KibanaSchema`

Reply:
Okay figured it out - exported variables need to be uppercase apparently!